package.json

```diff
@@ -59,7 +59,6 @@
     "react-dropzone": "^3.6.0",
     "react-ga": "1.4.1",
     "react-hot-loader": "1.3.0",
-    "react-loader": "2.4.0",
```

Comment:
I did that and it wasn't working, there was some unrelated dependency of searchkit that caused an error. So to fix it I deleted `node_modules`, ran `npm install`, then ran `npm install --save`, then did the `npm uninstall --save react-loader`. Not sure I'll ever understand npm